xbmc/windowing/X11/GLContextEGL.cpp

```diff
@@ -334,18 +334,25 @@ void CGLContextEGL::Detach()
 bool CGLContextEGL::IsSuitableVisual(XVisualInfo *vInfo)
 {
   EGLConfig config = GetEGLConfig(m_eglDisplay, vInfo);
+  return SuitableCheck(m_eglDisplay, config);
```

Comment:
`GetEGLConfig` now calls `SuitableCheck`, so this call is superfluous, making the whole `IsSuitableVisual` function superfluous since it then does nothing but call `GetEGLConfig`. I think you can remove it.

Reply:
Is removed and changed to "return config != EGL_NO_CONFIG;".